wbooks/src/interfaces/posts.ts

```diff
@@ -0,0 +1,12 @@
+export interface Post {
+  id: string;
+  source: string;
+  user: string;
+  avatar: string;
+  video?: string;
+}
+
+export type PostsRoutes = {
+  routes: undefined;
+  post: { post: Post };
```

Comment:
this is weird. Why post: {post: } Cant you change the name? and why routes is undefined?